electrum/plugins/trezor/qt.py

```diff
@@ -376,65 +385,65 @@ def show_expert_settings():
             rb_shamir.setVisible(True)
             rb_shamir_groups.setVisible(True)
         expert_button.clicked.connect(show_expert_settings)
-        vbox.addWidget(expert_button)
+        self.addWidget(expert_button)
 
         # passphrase
         passphrase_msg = WWLabel(PASSPHRASE_HELP_SHORT)
         passphrase_warning = WWLabel(PASSPHRASE_NOT_PIN)
         passphrase_warning.setStyleSheet("color: red")
-        cb_phrase = QCheckBox(_('Enable passphrases'))
-        cb_phrase.setChecked(False)
+        self.cb_phrase = QCheckBox(_('Enable passphrases'))
+        self.cb_phrase.setChecked(False)
         expert_vbox.addWidget(passphrase_msg)
         expert_vbox.addWidget(passphrase_warning)
-        expert_vbox.addWidget(cb_phrase)
+        expert_vbox.addWidget(self.cb_phrase)
 
         # ask for recovery type (random word order OR matrix)
-        bg_rectype = None
+        self.bg_rectype = None
         if method == TIM_RECOVER and model == '1':
             gb_rectype = QGroupBox()
             hbox_rectype = QHBoxLayout()
             gb_rectype.setLayout(hbox_rectype)
             expert_vbox.addWidget(gb_rectype)
             gb_rectype.setTitle(_("Select recovery type:"))
-            bg_rectype = QButtonGroup()
+            self.bg_rectype = QButtonGroup()
 
             rb1 = QRadioButton(gb_rectype)
             rb1.setText(_('Scrambled words'))
-            bg_rectype.addButton(rb1)
-            bg_rectype.setId(rb1, RecoveryDeviceType.ScrambledWords)
+            self.bg_rectype.addButton(rb1)
+            self.bg_rectype.setId(rb1, RecoveryDeviceType.ScrambledWords)
             hbox_rectype.addWidget(rb1)
             rb1.setChecked(True)
 
             rb2 = QRadioButton(gb_rectype)
             rb2.setText(_('Matrix'))
-            bg_rectype.addButton(rb2)
-            bg_rectype.setId(rb2, RecoveryDeviceType.Matrix)
+            self.bg_rectype.addButton(rb2)
+            self.bg_rectype.setId(rb2, RecoveryDeviceType.Matrix)
             hbox_rectype.addWidget(rb2)
 
         # no backup
-        cb_no_backup = None
+        self.cb_no_backup = None
         if method == TIM_NEW:
-            cb_no_backup = QCheckBox(f'''{_('Enable seedless mode')}''')
-            cb_no_backup.setChecked(False)
+            self.cb_no_backup = QCheckBox(f'''{_('Enable seedless mode')}''')
```

Comment:
Hmm, it's not related to the PR, but what is going on here... Looks like I wrote this some years ago >_<
Let's just do `QCheckBox(_('Enable seedless mode'))`